DatabaseTemplate/Modules/create-database-template.psm1

```diff
@@ -170,7 +170,7 @@ function Copy-InterchangeFiles {
         if ($includeAllInterchanges -or ($interchanges -contains (Get-XmlRoot $xmlFile.FullName).Name)) {
             $elapsed = Use-Stopwatch {
                 Write-Host "copy to $($directory.Name)\$($xmlFile.Name) " -NoNewline
-                Copy-Item -Path $xmlFile.FullName -Destination "$directory\$xmlFile"
+                Copy-Item -Path $xmlFile.FullName -Destination "$directory\$($xmlFile.Name)"
```

Comment:
In PS7 referencing a file returns the full path, so I use `.Name` here to get the file name without the path.